core/src/main/java/org/apache/iceberg/actions/BaseRewriteDataFilesAction.java

```diff
@@ -58,6 +59,7 @@
     extends BaseSnapshotUpdateAction<ThisT, RewriteDataFilesActionResult> {
 
   private static final Logger LOG = LoggerFactory.getLogger(BaseRewriteDataFilesAction.class);
+  private static final int PARQUET_MAGIC_BYTES = 4;
```

Comment:
This variable name doesn't correctly describe its contents because these are not the magic bytes, this is the magic size or length.

Reply:
Oh, yes, the name is incorrect... How about 'PARQUET_MAGIC_LEN' ?